src/Data/Csv/Parser.hs

```diff
@@ -43,6 +52,18 @@ import Control.Applicative ((<$>), (*>), (<*), pure)
 import Data.Monoid (mappend, mempty)
 #endif
 
+-- | Whether to escape fields with double quotes (@"@).
+data Escaping
+    = Escape
```

Comment:
So this is actually `MaybeEscape`, right? Escaping is allowed but not mandatory.

Reply:
Exactly. Would changing the name be more clear? (I guess the documentation explains it, but it might be more clear if the name was more precise.)

Reply:
Yes let’s change the name, it’s a small effort and it might save somebody some confusion later.